demo.html

```diff
@@ -19,24 +19,28 @@ <h1>Alerter Test</h1>
         <p>See the <a href="https://github.com/gosukiwi/alerter" 
             target="_blank">documentation</a> for more info!</p>
 
-        <script src="alerter.js"></script>
-        <script>(function () {
-            "use strict";
-            var i = 1;
+        <script src="alerter.min.js"></script>
```

Comment:
Let's use the development version for the demo so people can see the source code if curious